common/match/pool.ts

```diff
@@ -522,7 +522,7 @@ export async function getInterestConfirmationRate() {
         where: { status: InterestConfirmationStatus.CONFIRMED },
     });
 
-    return confirmedInterestConfirmations / totalInterestConfirmations;
+    return confirmedInterestConfirmations / totalInterestConfirmations || 0;
```

Comment:
We now run the integration tests without seeding the DB with test data (as the integration tests should be self-containting), so we now divide by zero here (which we did not handle properly